morse_translator.rb

```diff
@@ -40,4 +40,17 @@ def initialize
                     " " => " "}
   end
 
+  def match_text(text)
+    array_text = text.split('')
+     array_text.map do |char|
+      if @dictionary.include?(char)
+        @dictionary[char]
+      end
```

Comment:
I like the thinking here (trying to protect against a user who has entered a character not in our dictionary), but don't believe it was required based on the spec.